x/circuit/features/msg_trip.feature

```diff
@@ -0,0 +1,74 @@
+Feature: MsgTripCircuitBreaker
+
+	Circuit breaker can disable message execution:
+	- when the caller trips the circuitbreaker for a message or messages
+	- when the permissions are valid
+
+	Rule: the caller must have SUPER_ADMIN permission to trip the circuit
+		Example: caller is a super-admin
+			Given "acct1" has permission "LEVEL_SUPER_ADMIN"
+			When  attempts to disable msg execution
+				"""
+				{
+					"msg": "cosmos.bank.v1beta1.MsgSend"
+				}
+				"""
+			Then expect success
+		Example: caller has no permissions
+			Given "acct1" has no permissions
+			When "acct1" attempts to disable msg execution
+				"""
+				{
+					"msg": "cosmos.bank.v1beta1.MsgSend"
+				}
+				"""
+			Then expect an "unauthorized" error
+
+	Rule: the caller must have ALL_MSGS permission to trip the circuit
+		Example: caller has all msg permissions
+			Given "acct1" has permission "LEVEL_ALL_MSGS"
+			When "acct1" attempts to disable msg execution
+				"""
+				{
+					"msg": "cosmos.bank.v1beta1.MsgSend"
+				}
+				"""
+			Then expect an "unauthorized" error
+		Example: caller has LEVEL_SOME_MSGS permissions
+			Given "acct1" has permission "LEVEL_SOME_MSGS"
+			When "acct1" attempts to disable msg execution
+				"""
+				{
+					"msg": "cosmos.bank.v1beta1.MsgSend"
+				}
+				"""
+			Then expect an "unauthorized" error
+		Example: caller has no permissions
+			Given "acct1" has no permissions
+			When "acct1" attempts to disable msg execution
+				"""
+				{
+					"msg": "cosmos.bank.v1beta1.MsgSend"
+				}
+				"""
+			Then expect an "unauthorized" error
+
+	Rule: the caller must have LEVEL_SOME_MSGS permission to trip the circuit
```

Comment:
For `LEVEL_SOME_MSGS` we want a rule that specifies that they must have permissions to trip all the msg types they're trying to disable. So we'd want a case where they try to trip some types they aren't authorized to trip and it should fail